impl/src/main/java/dao/AnswerDao.java

```diff
@@ -19,18 +20,22 @@
 
     @Query("SELECT answer.id, answer.user_id, answer.answer, answer.created_at, answer.accepted, answer.title, answer.votes , answer.slack_id, \"user\".\"name\"  as created_by from answer \n" +
         "INNER JOIN \"user\" on \"user\".id = answer.user_id \n" +
-        "where question_id=:questionId\n")
+        "where question_id=:questionId order by answer.accepted desc, answer.votes desc, answer.created_at desc")
```

Comment:
Nitpick, mixed uppercase and lowercase keywords in the SQL syntax.

Reply:
Ain't no nitpick, anything but upper case keywords is heresy :D